.github/workflows/test_in_devenv.yml

```diff
@@ -219,11 +219,11 @@ jobs:
               echo "::error file=test_in_devenv.yml::Python tests failed with status $pytest_status."
               exit 1
             fi
-            CUDAQ_PYTEST_EAGER_MODE=OFF python3 -m pytest -v python/tests/ \
+            CUDAQ_PYTEST_EAGER_MODE=ON python3 -m pytest -v python/tests/ \
```

Comment:
Do some of the tests not support MLIR mode yet?

Reply:
It's the other way round, some tests are only for MLIR mode. 
Prior to this change, eager mode was default, and we ran all tests in both modes. Now that MLIR mode is default, I changed this environment variable to 'ON' to run other mode.